LocationManager/INTULocationManager/INTULocationManager.h

```diff
@@ -125,6 +130,33 @@ __INTU_ASSUME_NONNULL_BEGIN
 /** Immediately cancels the location request (or subscription) with the given requestID (if it exists), without executing the original request block. */
 - (void)cancelLocationRequest:(INTULocationRequestID)requestID;
 
+#pragma mark Heading Requests
+
+/** 
+ Whether or not the CLLocationManager is currently sending heading updates. 
+ */
+@property (nonatomic, assign, readonly) BOOL isUpdatingHeading;
+
+/**
+ * Specifies the minimum amount of change in degrees needed for a heading service update. Observers will not be notified of updates less than the stated filter value.
+ *
+ * @discussion The default value for this is 1.
+ */
+@property (nonatomic) INTUHeadingFilterAccuracy headingFilter;
```

Comment:
Agreed; I'll need to investigate this one a bit more to see what would make sense for instantiating the requests with a given accuracy AND filter value (for locations at least) since it adds another initializer to the mix and is kind of confusing. Any thoughts on how you'd like this implemented?